packages/cli/README.md

```diff
@@ -181,10 +193,10 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
Would be good to confirm that this command actually uses `--key`.  It seems like it totally could, or maybe not since the command says **did** in the name, maybe it's using a globally configured DID instance and ignores the `--key` flag passed in?  If --key is ignored here it could actually be quite confusing for users as it *could* make sense to provide a custom key to do the signature with.

Reply:
This does require an authenticated did. if a key hasn't been provided previously a key will be required.